example/tests/unit/test_default_drf_serializers.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+from django.urls import reverse
```

Comment:
Imports are in invalid order according to isort.

See https://travis-ci.org/django-json-api/django-rest-framework-json-api/jobs/440153239#L936